src/utils/__tests__/__snapshots__/getConfig.test.js.snap

```diff
@@ -242,3 +242,182 @@ Object {
   "target": "webworker",
 }
 `;
+
+exports[`creates config: ios config with hot reloading disabled 1`] = `
```

Comment:
This snapshot is too big. Please use snapshot-diff and compare default config to this